docs/resources/azure_monitor_activity_log_alerts.md.erb

```diff
@@ -0,0 +1,63 @@
+---
+title: About the azure_monitor_activity_log_alerts Resource
+platform: azure
+---
+
+# azure\_monitor\_activity\_log\_alerts
+
+Use the `azure_monitor_activity_log_alerts` InSpec audit resource to verify an Activity Log
+Alert exists.
+
+<br />
+
+## Syntax
+
+An `azure_monitor_activity_log_alerts` resource block identifies a Activity Log Alerts by name.
+
+    describe azure_monitor_activity_log_alerts do
+      ...
+    end
+<br />
+
+## Examples
+
+### Test an example resource has an Activity Log Alert
+
+    describe azure_monitor_activity_log_alerts do
+      its('names') { should include('example-log-alert') }
+    end
+
+<br />
+
+## Properties
+
+* `names`
+
+### names
+
+The name of the Activity Log Alert
+
+    its('names') { should include('example-log-alert') }
+
+## Matchers
+
+This InSpec audit resource has the following special matchers. For a full list
+of available matchers, please visit our [Universal Matchers page](https://www.inspec.io/docs/reference/matchers/).
+
+### exists
+
+The control will pass if the resource returns a result. Use `should_not` if you expect zero matches.
+
+    # example should exist
+    describe azure_monitor_activity_log_alerts do
+      its('names') { should include('example-log-alert') }
+    end
+
+    # this activity log should not exist
+    describe azure_monitor_activity_log_alerts do
+      its('names') { should_not include('example-log-alert') }
+    end
```

Comment:
It would be more clear to provide different log alert names in the above examples.